tools/distrib/check_boringssl_prefix_symbol.sh

```diff
@@ -22,6 +22,6 @@ cd ../../third_party/boringssl-with-bazel
 BORINGSSL_COMMIT=$(git rev-parse HEAD)
 PREFIX_SYMBOLS_COMMIT=$(cat ../../src/boringssl/boringssl_prefix_symbols.h | head -n1 | awk '{print $NF}')
 
-[ $BORINGSSL_COMMIT == $PREFIX_SYMBOLS_COMMIT ] || { echo "The BoringSSL commit does not match the commit of the prefix symbols (src/boringssl/boringssl_prefix_symbols.h). Run tools/distrib/regenerate_boringssl_prefix_symbols.sh to update the prefix symbols." ; exit 1 ; }
+[ $BORINGSSL_COMMIT == $PREFIX_SYMBOLS_COMMIT ] || { echo "The BoringSSL commit does not match the commit of the prefix symbols (src/boringssl/boringssl_prefix_symbols.h). Run tools/distrib/generate_boringssl_prefix_header.sh to update the prefix symbols." ; exit 1 ; }
```

Comment:
Looks like the filename is being fixed here, but I'm not sure why the wrong file name wasn't reported by any of our tests?

Reply:
As far as I can tell this is an error message pointing a specific file as a hint for mismatched versions as to how to fix the issue.  Maybe there aren't any tests for this?  Or if you could give me a pointer to where you think the tests might be I'll take a look.